test/constraints/selector.jl

```diff
@@ -174,6 +174,18 @@ function test_nested_atmost_reformulation()
         DP._indicator_to_binary(model)[y[3]]
 end
 
+function test_extension_variables()
+    model = GDPModel{MyModel, MyVarRef, MyConRef}()
+    @variable(model, y[1:2], Logical(MyVar), start = true)
+    cref = LogicalConstraintRef(model, LogicalConstraintIndex(1))
+    @test @constraint(model, y in AtLeast(1)) == cref
+    @test reformulate_model(model, DummyReformulation()) isa Nothing
```

Comment:
With having BigM as the default, you can remove the DummyReformulation here unless we want to keep it.

Reply:
We can keep it as an extra Reformulation method to test extension methods